From dfa6daf112dfae5b6de4dd658b1acc0e94cf7294 Mon Sep 17 00:00:00 2001
From: Guillaume Aubertin <guillaume.aubertin@intel.com>
Date: Thu, 30 Oct 2014 18:50:16 +0100
Subject: [PATCH 2/3] bcmdhd: disabling ACPI PM for PCIe

disabling PCIe power management as long as there is only
a BCM43241 ACPI device entry. PCI needs its own PS0 and PS3
methods.

Change-Id: I9172805481f79fcc35ab90e6b03f448a12433bb5
Signed-off-by: Guillaume Aubertin <guillaume.aubertin@intel.com>
---
 drivers/net/wireless/bcmdhd/dhd_linux_platdev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/bcmdhd/dhd_linux_platdev.c b/drivers/net/wireless/bcmdhd/dhd_linux_platdev.c
index e56cbf11187a..86e2d2cf188a 100644
--- a/drivers/net/wireless/bcmdhd/dhd_linux_platdev.c
+++ b/drivers/net/wireless/bcmdhd/dhd_linux_platdev.c
@@ -393,7 +393,7 @@ int wifi_platform_set_power(wifi_adapter_info_t *adapter, bool on, unsigned long
 		err = plat_data->set_power(on);
 	}
 
-#ifdef CONFIG_ACPI
+#if defined(CONFIG_ACPI) && defined(BCMSDIO)
 	dev = bus_find_device(&platform_bus_type, NULL, bcm_acpi_id, wifi_acpi_match);
 	pdev = to_platform_device(dev);
 	if (ACPI_HANDLE(&pdev->dev)) {
-- 
2.0.3

